go.mod

```diff
@@ -0,0 +1,66 @@
+module go-ipld-prolly-trees
```

Comment:
Please, consider naming the module after the GitHub URL to make it go-getable.

Reply:
Agreed, would nbe good to check that we can add this module easily via go-get.

Reply:
Fixed. Use `go env -w GOPROXY="direct"` if you have trouble installing the latest commit.